specs/phase0/p2p-interface.md

```diff
@@ -229,7 +239,7 @@ where `base64` is the [URL-safe base64 alphabet](https://tools.ietf.org/html/rfc
 
 The payload is carried in the `data` field of a gossipsub message, and varies depending on the topic:
 
-| Topic                                          | Message Type            |
+| Name                                           | Message Type            |
```

Comment:
It seems we are still using `TopicName` to describe the field in the following. (Section `Global topics
` and `Attestation subnets`)

Reply:
updated this in #1652